EIPS/eip-4626.md

```diff
@@ -106,7 +106,9 @@ MUST NOT show any variations depending on the caller.
 
 MUST NOT reflect slippage or other on-chain conditions, when performing the actual exchange.
 
-MUST NOT revert.
+MUST NOT revert unless due to integer overflow caused by an unreasonably large input.
+
+MUST round down, in favor of the user.
```

Comment:
i'm still uncomfortable about this, its usually best practice to give users numbers rounded in favor of the vault not themselves

Reply:
conversion functions should be "favor" agnostic imo

Reply:
agreed, correcting